server/controllers/userController.js

```diff
@@ -25,13 +25,27 @@ class UserController {
       const newUser = await User.create({ firstName, lastName, email, password: bcrypt.hashSync(password, 10), avatarUrl, verified: false })
       if (newUser) {
         const { id } = newUser.dataValues
-        const token = createToken({ id }, '24h');
+        const token = createToken({ id, email }, '24h');
```

Comment:
why are you adding email here? only `id` should be here

Reply:
fixed